RecoTracker/CkfPattern/src/CkfTrajectoryBuilder.cc

```diff
@@ -48,6 +50,8 @@ void CkfTrajectoryBuilder::fillPSetDescription(edm::ParameterSetDescription& iDe
   BaseCkfTrajectoryBuilder::fillPSetDescription(iDesc);
   iDesc.add<int>("maxCand", 5);
   iDesc.add<double>("lostHitPenalty", 30.);
+  iDesc.add<double>("foundHitBonus", 10.);
+  iDesc.add<int>("minHitForDoubleBonus", 8);
```

Comment:
do I understand correctly that with `"foundHitBonus", 0.` there will be no differences?

Was the non-zero value here added just to run the tests?
In the email thread I was proposing to make a technical PR.

Reply:
0 will be NoOp yes.
I can change default to NoOp if requested.

Reply:
> 0 will be NoOp yes. I can change default to NoOp if requested.

I may be more comfortable with a physics change after seeing some more complicated electron sources (including fakes). ttbar could be a good test.